### DISCUSSION TITLE
Disconnecting device from whatsapp

### DISCUSSION THREAD

**Discussion by ghost** [devlikeapro/waha#147] [Q&A] [answered]
When I end a session, why doesn't WAHA automatically disconnect my device from WhatsApp?
It only shows I'm inactive but it doesn't disconnect the device in WhatsApp.
I would greatly appreciate any help.

Thanks

**Comment by allburov** [comment, marked as answer]
Hi! What do you mean by "end a session"? `POST /logout` endpoint, right? Yes, it actually removes only local information, but not the one on phone.
It's possible to handle both tho, I've created an issue so we won't forget about the request https://github.com/devlikeapro/whatsapp-http-api/issues/148

**Comment by ghost** [comment]
Thanks

**Comment by devlikepro** [comment]
Fixed in `2024.10.9`

When a session in `WORKING` status on `POST /api/sessions/{NAME}/logout`  request it'll try to remove the device from **Connected Devices** list in the app.

https://waha.devlike.pro/docs/how-to/sessions/#logout-session

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
